source/_docs/unsupported-modules-plugins.md

```diff
@@ -277,6 +277,14 @@ For more details, see [SERVER_NAME and SERVER_PORT on Pantheon](/docs/server_nam
 **Issue**: The Instashow plugin relies on query parameters that are not compatible with Pantheon's Edge Cache. See [PANTHEON_STRIPPED](https://pantheon.io/docs/pantheon_stripped/) for more information. This inhibits the ability to set the authorization token required to make the plugin function.
 <hr>
 
+### [TubePress](https://wordpress.org/plugins/tubepress/)
+**Issue**: TubePress will not work if you are using PHP 5. The server requirements for TubePress state that the PHP APC extension should be disabled. Pantheon provides APC by default across all plans and it is not possible to disable.
+
+**Solution**: Upgrade your site's PHP version to PHP 7 by enabling that option in a pantheon.yml file.
+
+See [Upgrade PHP Versions](https://pantheon.io/docs/php-versions/) and [The pantheon.yml Configuration File](https://pantheon.io/docs/pantheon-yml/) for more information.
+<hr>
```

Comment:
@rachelwhitton the server requirements for this particular plugin only mention PHP 5.3.29 without the APC extension. I was unable to find anywhere or anyone that mentioned it would work with PHP 7.

http://support.tubepress.com/customer/en/portal/articles/1994883-server-requirements
http://community.tubepress.com/topic/5779-tubepress-418-white-screen-of-death-with-php-5437/

I was only able to find references to issues with PHP 5.3.29 and 5.4.37 having problems with APC.

This particular customer site was white screen with this plugin enabled until we switched to PHP 7.

Case number: 79812